app/Http/Controllers/AntelopeActivity.php

```diff
@@ -173,10 +216,12 @@ public function passActivityData()
         'activity.patrol_end_date',
         'activity.start_time',
         'activity.end_time',
+        'activity.total_time',
```

Comment:
Do the calculation in the backend here, ->AddColumn and then call AntelopeCalculate, there should already be a function made for this.

Reply:
Ended up leaving it on the initial submit, so it can just be pulled as a string and does not need to be calculated every single time. When the edit patrol log update comes, it will need to be recalculated again then, and resaved as a string.